.changeset/config.json

```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@2.3.0/schema.json",
+  "changelog": "@changesets/cli/changelog",
```

Comment:
After reading your message about CI complaining this package was missing, I checked their docs at https://github.com/changesets/changesets/blob/main/docs/config-file-options.md#changelog-false-or-a-path, It seems this should be `@changesets/changelog-git` or `@changesets/changelog-github`.
Additionally, we could even set it to `false` if its a pain, because we never had changelog and even though is a nice feature, we could skip it (if it really procudes pain, you know).